### DISCUSSION TITLE
iBAQ

### DISCUSSION THREAD

**Discussion by haiyan-MS** [vdemichev/DiaNN#709] [Q&A]
HI 
I was wondering if we can get iBAQ value some how? All I need is observable number of peptides for given protein/gene. 
Also, is there a way to get parent M/Z? 

Thank you for thie wonderful software.

Haiyan

**Comment by vdemichev** [comment]
Hi Haiyan,

Not out of the box, but I guess there are likely some R packages that should do that? Or can write a script that loads the FASTA in R, there are packages for this. 

Best,
Vadim

**Comment by camoin** [comment]
﻿You can have a look at DIAgui package on GitHubLucLucLe 22 mai 2023 à 23:50, haiyan-MS ***@***.***> a écrit :﻿
HI
I was wondering if we can get iBAQ value some how? All I need is observable number of peptides for given protein/gene.
Also, is there a way to get parent M/Z?
Thank you for thie wonderful software.
Haiyan
